README.md

```diff
@@ -1 +1,12 @@
-# opensuse-jeptalon-base
\ No newline at end of file
+# openSUSE and JepTalon image
+
+This project builds on the openSUSE Python2 image [here](https://github.com/CAFapi/opensuse-python2-image), Talon and Jep are then installed on the image. This image can be used as a base by projects which require Python2, Talon and Jep.
+
+### Tini
+[Tini](https://github.com/krallin/tini) is pre-installed in the container.  If the image entrypoint is not overwritten then it will be automatically used.
+
```

Comment:
If you are copying over the features from the python2 image readme can you also include the postgreSQL client section?

Reply:
I have added this section along with some other feature information coming from the base image in this pull request: https://github.com/CAFapi/opensuse-jeptalon-base/pull/2

Reply:
PR Merged.